docs/source/cuda/examples.rst

```diff
@@ -7,19 +7,25 @@ Examples
 
 Matrix multiplication
 =====================
+First, import the modules needed for this example:
 
-Here is a naive implementation of matrix multiplication using a CUDA kernel::
+.. literalinclude:: ../../../numba/cuda/tests/doc_examples/test_matmul.py
+   :language: python
+   :caption: from ``test_ex_matmul`` in ``numba/cuda/tests/doc_example/test_matmul.py``
```

Comment:
```suggestion
   :caption: from ``test_ex_matmul`` in ``numba/cuda/tests/doc_examples/test_matmul.py``
```

Reply:
When I clicked "commit suggestion", it auto-resolved this. Instead should I be clicking "add suggestion to batch", or just commenting on them/not doing anything?

Reply:
I remarked it as unresolved to make the above comment more visible.